docs/using/tutorial/Vagrantfile.template

```diff
@@ -10,6 +10,41 @@ VAGRANTFILE_API_VERSION = "2"
 
 ENV['VAGRANT_DEFAULT_PROVIDER'] = 'virtualbox'
 
+$bootstrap_common = <<EOF
+mkdir -p /etc/flocker
+cp /home/vagrant/credentials/cluster.crt /etc/flocker/cluster.crt
+EOF
+
+$bootstrap_control = <<EOF
+cp /home/vagrant/credentials/control-172.16.255.250.crt /etc/flocker/control-service.crt
```

Comment:
Again script arguments, it would be nice to not repeat the IP address all over this file.